docs/release-testing.md

```diff
@@ -0,0 +1,7 @@
+With every rancher release, a new operator version may or may not be released. This document provides a reference to what tests should be run for both the cases.
+
+`P0` suite must be run to test every new rancher release. This suite tests provisioning and importing clusters, k8s upgrade of the control plane and nodes, and CRUD operations of the nodegroup/nodepool.
```

Comment:
```suggestion
`P0` suite must be run to test every new rancher release, preferably from Rancher Dashboard. This suite tests provisioning and importing clusters, k8s upgrade of the control plane and nodes, and CRUD operations of the nodegroup/nodepool.
```

Reply:
I am not sure I understand this suggestion. Why specifically Rancher Dashboard?

Reply:
Are these instructions specific to only automated tests from this repo ? In that case please ignore the suggestion.

Reply:
They are general instructions of what tests should be run for release testing.

Reply:
> When you say Rancher Dashboard, do you mean the tests should be run manually via UI?

Yes, not to miss running P0 tests from UI in release testing.